coeus-impl/src/main/resources/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetSettings.xml

```diff
@@ -30,7 +30,7 @@
 				<property name="items">
 					<list>
 						<bean parent="Uif-PrimaryActionButton" p:actionLabel="Apply Changes" p:methodToCall="saveBudgetSettings"
-							p:order="10" p:refreshId = "PropBudget-BudgetSettings-Dialog" p:dialogDismissOption=""
+							p:order="10" p:dialogDismissOption="REQUEST"
```

Comment:
I set the dialogDismissOption so that error messages will be displayed in the dialog. Doing this means those error either won't appear at all or will appear on the main page instead of in the settings.